web/src/lib/stores.ts

```diff
@@ -11,3 +11,9 @@ export const Info: Writable<string | null | {message: string, duration: number,
 export const Error: Writable<string | null> = writable(null);
 
 export const BTC2USD: Writable<number | null> = writable(null);
+
+type showLoginInfoType = {
+    opened: boolean;
+    callbackFunc?: object | null;
+};
+export const showLoginModal: Writable<showLoginInfoType | null> = writable(null);
```

Comment:
I think this should also not be nullable - we want this store to keep the current state of the log in modal, which might mean the modal is not visible, that nobody requested it to be shown, etc ... but that doesn't mean that the state itself is null - the state is still a valid state.

Reply:
Yeah, you're right. This is a remaining of a previous way of doing things. Not needed now.